src/Tools/ExternalAccess/Razor/RazorSemanticTokensAccessor.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis.LanguageServer.Handler.SemanticTokens;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Razor
+{
+    internal class RazorSemanticTokensAccessor
+    {
+        public static ImmutableArray<string> RoslynTokenTypes => SemanticTokensHelpers.AllTokenTypes;
```

Comment:
One bit we'll want to keep in mind is that once Razor consumes this & wants to insert into VS4Mac we'll need to ensure that the corresponding Roslyn bits are there first. Just bringing it up because there's a lot of movement in that space recently 😄

Reply:
No regular schedule, no. The build validation on VS Mac is good though, and will report a failure if there is an incompatibility between inserted versions of roslyn and razor.